serve/graph/schema/filter/transaction_filter.graphql

```diff
@@ -65,6 +65,54 @@ input TransactionFilter {
   `success` enables you to filter between successful and unsuccessful transactions.
   """
   success: Boolean
+
+  """
+  `events` are the events that the transaction has emitted.
+  `events` filters transactions with specific events.
+  """
+  events: [TransactionEventInput!]
+}
+
+"""
+Transaction's event to filter transactions.
+"TransactionEventInput" can be configured as a filter with a transaction event's `type` and `pkg_path` and `func`, and `attrs`.
+"""
+input TransactionEventInput {
+  """
+  `type` is the type of transaction event emitted.
+  """
+  type: String
+
+  """
+  `pkg_path` is the path to the package that emitted the event.
+  """
+  pkg_path: String
```

Comment:
This filter is redundant, isn't it? we can filter Transactions by its pkg_path, depending on the transaction type.

Reply:
If you examine a single event, it may be a duplicate. 
However, The fields in events can be filtered as an array.

I think it's fine to be able to set per-event package paths.